images/libvirt-kubevirt/libvirtd.sh

```diff
@@ -45,6 +45,8 @@ mkdir -p /var/log/kubevirt
 touch /var/log/kubevirt/qemu-kube.log
 chown qemu:qemu /var/log/kubevirt/qemu-kube.log
 
+tail -F /var/log/kubevirt/qemu-kube.log >&2 &
```

Comment:
I think we need a command grouping here, otherwise it can mess up our logs, since tailf and libvirtd will write to stdout at the same time ...